regression: unresolvable imports

PR #6875 added code in `tfjs-layers/src/layers/preprocessing/center_crop.ts`

```js
import {resizeBilinear} from '@tensorflow/tfjs-core/ops/image/resize_bilinear';
import {cropAndResize} from '@tensorflow/tfjs-core/ops/image/crop_and_resize';
```

how is this import supposed to be resolved?  
i cannot get a clean build of `tfjs` due to this. and it goes against all other import syntax used.

```log
src/tfjs-layers/src/layers/preprocessing/center_crop.ts:13:30 - error TS2307: Cannot find module '@tensorflow/tfjs-core/ops/image/resize_bilinear' or its corresponding type declarations.
```

cc @AdamLang96 @mattsoulanille

--- a/tfjs-layers/src/layers/preprocessing/center_crop.ts
+++ b/tfjs-layers/src/layers/preprocessing/center_crop.ts
@@ -8,14 +8,14 @@
  * =============================================================================
  */
 
-import {serialization,DataType,unstack,stack,tensor,Tensor,Tensor1D,Tensor2D, Tensor3D, Tensor4D, tidy, range} from '@tensorflow/tfjs-core';
+import {serialization,DataType,unstack,stack,tensor,Tensor,Tensor1D,Tensor2D, Tensor3D, Tensor4D, tidy, range, image} from '@tensorflow/tfjs-core';
 import {getExactlyOneShape, getExactlyOneTensor} from '../../utils/types_utils';
-import {resizeBilinear} from '@tensorflow/tfjs-core/ops/image/resize_bilinear';
-import {cropAndResize} from '@tensorflow/tfjs-core/ops/image/crop_and_resize';
 import {LayerArgs, Layer} from '../../engine/topology';
 import {Kwargs} from '../../types';
 import {Shape} from '../../keras_format/common';
 import * as K from '../../backend/tfjs_backend';
+
+const {resizeBilinear, cropAndResize} = image;
 
 export declare interface CenterCropArgs extends LayerArgs{
   height: number;

